openmc/mesh.py

```diff
@@ -1137,6 +1137,70 @@ def from_hdf5(cls, group):
 
         return mesh
 
+    @classmethod
+    def from_domain(
+        cls,
+        domain,
+        dimension=[100, 100, 100],
+        mesh_id=None,
+        phi_grid=[0.0, 2*pi],
+        name=''
+    ):
+        """Create mesh from an existing openmc cell, region, universe or
+        geometry by making use of the objects bounding box property. phi_grid
+        is set to have a range of 0 to 2π by default.
+
+        Parameters
+        ----------
+        domain : {openmc.Cell, openmc.Region, openmc.Universe, openmc.Geometry}
+            The object passed in will be used as a template for this mesh. The
+            bounding box of the property of the object passed will be used to
+            set the r_grid, z_grid ranges.
+        dimension : Iterable of int
+            The number of equally spaced mesh cells in each direction (r_grid,
+            phi_grid, z_grid)
+        mesh_id : int
+            Unique identifier for the mesh
+        phi_grid : numpy.ndarray
+            1-D array of mesh boundary points along the phi-axis in radians.
```

Comment:
I think this is not correct as this phi_grid argument is not a 1-D of mesh boundary points but rather the mesh bounds.

Reply:
thanks, I've updated the doc string wordin